platform-sdk/consensus-otter-tests/src/test/java/org/hiero/otter/test/DocExamplesTest.java

```diff
@@ -34,10 +33,8 @@ void testConsensus(@NonNull final TestEnvironment env) throws InterruptedExcepti
         // 4. Wait 30 seconds while the network is running
         timeManager.waitFor(Duration.ofSeconds(30));
 
-        // 5. Verify consensus was reached
-        assertThat(network.getConsensusResults())
-                .haveEqualCommonRounds()
-                .haveMaxDifferenceInLastRoundNum(withPercentage(5));
+        // 5. Verify consensus was reached and advanced past round 2
+        assertThat(network.getConsensusResults()).haveEqualCommonRounds().haveAdvancedSinceRound(2);
```

Comment:
Please also update the code example in the documentation (`README.md` and `getting-started.md`)